File: Serene/Serene.Script/Imports/FormContexts/Meeting.MeetingForm.cs

namespace Serene.Meeting
{
    using Serenity;
    using Serenity.ComponentModel;
    using System;
    using System.Collections;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.Runtime.CompilerServices;
    using Serene.Northwind;

    public partial class MeetingForm : PrefixedContext
    {
        public MeetingForm(string idPrefix) : base(idPrefix) {}
    
        public StringEditor MeetingGuid { get { return ById<StringEditor>("MeetingGuid"); } }
        public StringEditor MeetingName { get { return ById<StringEditor>("MeetingName"); } }
        public IntegerEditor MeetingTypeId { get { return ById<IntegerEditor>("MeetingTypeId"); } }
        public DateEditor StartDate { get { return ById<DateEditor>("StartDate"); } }
        public DateEditor EndDate { get { return ById<DateEditor>("EndDate"); } }
        public IntegerEditor InsertUserId { get { return ById<IntegerEditor>("InsertUserId"); } }
        public IntegerEditor InsertDate { get { return ById<IntegerEditor>("InsertDate"); } }
        public StringEditor IsActive { get { return ById<StringEditor>("IsActive"); } }
        public IntegerEditor UpdateUserId { get { return ById<IntegerEditor>("UpdateUserId"); } }
        public IntegerEditor UpdateDate { get { return ById<IntegerEditor>("UpdateDate"); } }
        public IntegerEditor LocationId { get { return ById<IntegerEditor>("LocationId"); } }
        public IntegerEditor OrganizationId { get { return ById<IntegerEditor>("OrganizationId"); } }
        public IntegerEditor OrganizerContactId { get { return ById<IntegerEditor>("OrganizerContactId"); } }
        public IntegerEditor ReporterContactId { get { return ById<IntegerEditor>("ReporterContactId"); } }
    }
}

File: Serene/Serene.Web/Modules/Common/Imports/ServerTypings/Meeting.MeetingForm.ts
namespace Serene.Meeting {
    export class MeetingForm extends Serenity.PrefixedContext {
        static formKey = 'Meeting.Meeting';

    }

    export interface MeetingForm {
        MeetingName: Serenity.StringEditor;
        MeetingTypeId: Serenity.LookupEditor;
        MeetingNumber: Serenity.StringEditor;
        StartDate: Serenity.DateTimeEditor;
        EndDate: Serenity.DateTimeEditor;
        LocationId: Serenity.LookupEditor;
        UnitId: Organization.BusinessUnitEditor;
        OrganizerContactId: Serenity.LookupEditor;
        ReporterContactId: Serenity.LookupEditor;
        AttendeeList: MeetingAttendeeEditor;
    }

    [['MeetingName', () => Serenity.StringEditor], ['MeetingTypeId', () => Serenity.LookupEditor], ['MeetingNumber', () => Serenity.StringEditor], ['StartDate', () => Serenity.DateTimeEditor], ['EndDate', () => Serenity.DateTimeEditor], ['LocationId', () => Serenity.LookupEditor], ['UnitId', () => Organization.BusinessUnitEditor], ['OrganizerContactId', () => Serenity.LookupEditor], ['ReporterContactId', () => Serenity.LookupEditor], ['AttendeeList', () => MeetingAttendeeEditor]].forEach(x => Object.defineProperty(MeetingForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}

